Gemfile

```diff
@@ -16,3 +16,4 @@ gem 'faker'
 gem 'systemd-journal'
 gem 'rubocop'
 gem 'sinatra-base'
+gem 'mock_redis'
```

Comment:
I was only worried about unit testing for this iteration. When I write integration tests we'll use the real deal.